src/components/bookmark/bookmark.tsx

```diff
@@ -0,0 +1,30 @@
+import clsx from 'clsx';
+import { Size } from '../../types/common';
+
+type BookmarkProps = {
+  className: string;
+  isBookmarked: boolean;
+}
+
+const iconSizes: {[key: string]: Size} = {
+  'default': {
```

Comment:
У объектов ключ строка всегда можно просто писать 
```suggestion
  default: {
```